docs/features/github/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 1
+title: GitHub | Overview
+hide_table_of_contents: true
+hide_title: true
+---
+
+import DocsLinkCard from "@site/src/components/LinkCard";
+
+export const tutorials = [
+    {
+        title: 'Automated Authorization Pull Requests',
+        description: 'Automatically create pull requests to resolve any drifts detected in your cluster',
+        url: '/features/github/tutorials/automated-pull-requests'
+    },
+];
+
+
+# GitHub
+
+Otterize can create just-in-time AWS IAM roles and policies for your workloads running on EKS Kubernetes clusters, greatly simplifying the lifecycle of managing IAM roles and policies.
+
+### Tutorials
+
+To learn how to leverage Otterize's continuous monitoring of your cluster's access to detect and resolve any drifts.
+<DocsLinkCard items={tutorials} colSize={"sm"}/>
+
+### How does Otterize work with GitHub?
+
+After installing Otterize in your cluster, it immediately begins monitoring both incoming and outgoing network traffic to your pods and the public internet via the network mapper feature (see Network Mapper for more details). Additionally, Otterize can track activity across various resources, including Postgres, AWS, Kafka, and more. This tracking helps Otterize create a detailed map of interactions and relationships.
+
+By leveraging ClientIntents, Otterize simplifies the application and enforcement of access rights, thereby enhancing cluster security. Once ClientIntents are defined and stored in a GitHub repository, Otterize compares these intended access patterns against actual usage within the cluster. Discrepancies trigger a pull request to update the ClientIntent definitions to reflect current usage. This process enables development teams to either accept the suggested changes or adjust them based on newly observed behaviors.
```

Comment:
Can this be split into multiple paragraphs? It's a bit of a mouthful.
For example:

Once you have declared ClientIntents and enabled enforcement, anything missing from the ClientIntents will be blocked. Fortunately, Otterize is able to compare the ClientIntents in your GitHub-hosted repository with the actual traffic in your cluster.

If a difference is detected, Otterize will submit a Pull Request, updating the ClientIntents in your Git with the ClientIntents you should have, based on your traffic. If this connection is desirable, a developer can approve the Pull Request. This significantly reduces friction for developers, by providing them with the right answer when a problem occurs, and doing it all in Git.

You can also use the GitHub integration in shadow mode, when Otterize is not enforcing. In this case, Otterize will help you gradually build up the ClientIntents necessary to allow intended access, without risking connections being blocked. Once you no longer get new pull requests for new connections, you can feel safe in knowing you have all the necessary ClientIntents declared, and activate enforcement.

Reply:
Done, please take a look at the last commit.